lib/wickedbase.pm

```diff
@@ -984,6 +985,12 @@ sub post_run {
     $self->check_logs() unless $self->{skip_check_logs_on_post_run};
     $self->check_coredump();
     $self->upload_wicked_logs('post');
+
+    my $time_since_barrier_wait = time - ($self->{fix_barrier_wait} // 0);
```

Comment:
first of all I am not fully understanding what is going on here , so please explain and I think it will make sense to put at least part of your explanation here so future us or someone who will have luck to support this code after us will have some hints

Reply:
ack